src/main/java/Nocturne.java

```diff
@@ -0,0 +1,94 @@
+import java.util.Scanner;
+
+public class Nocturne {
+    public static void main(String[] args) throws NocturneException {
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        String input;
+        Scanner in = new Scanner(System.in);
+        System.out.println("Good evening. I'm Nocturne.");
+        System.out.println("What ails you on this fine day?");
+
+        input = in.nextLine();
+        while (!input.equals("bye")) {
+            String[] commandCheck = input.split(" ");
```

Comment:
note that plural form should be used on names representing a collection of objects. might want to change this to a different name instead to for better readability - perhaps something like commandParts/commandTypes or other appropriate names!